CHANGELOG.md

```diff
@@ -27,6 +27,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed a bug in `WandbWorkspace.step_finished()` which led to a `RuntimeError` sometimes while
   caching the result of a step.
 
+### Added
+
+- Added the ability to train straight `Model` instead of just `Lazy[Model]`
```

Comment:
Needs to be moved up to unreleased section.

Reply:
done